dirs.proj

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this file should be named "build.proj" to align with the naming used in "corefx".

"dirs.proj" files traditionally are supposed to be proj files that include other proj files - think like a .sln file that just aggregates other projects.

Here's some examples:
https://github.com/Microsoft/cpprestsdk/blob/3070ca21ed665a5aeb7d0cfe28f7f21d6c062a92/Release/tests/functional/dirs.proj
https://github.com/Microsoft/PTVS/blob/b08cc4efea4b6dd8f8db75b2bc8dd6513c41079f/Python/Setup/dirs.proj
https://github.com/Microsoft/nodejstools/blob/d966849ae462c543313291ad20d419a27c8e1fc7/dirs.proj